set_ce_status.py

```diff
@@ -0,0 +1,136 @@
+######################################################
+# set_ce_status.py
+# -----------
+# Author: Stefanie Langrock
+#         <s.langrock@qmul..ac.uk>
+#
+# Description:
+#    Script to update CE status on CouchDB
+#
+# Can exclude or allow CEs according to the user input 
+# from the command line. Extracts a list of all CEs 
+# available to snoplus.snolab.ca and compares it to the
+# CEs listed in the database. Adds CEs to the database 
+# if they are not included.  
+# 
+######################################################
+
+import os
+import couchdb
+import getpass
+import optparse
+
+from GangaSNOplus.Lib.Applications import job_tools
+
+def set_ce_permissions(ce_list, si=None):
+
+    #access the server using the admin log in
+    server = couchdb.Server("http://snoplus.cpp.ualberta.ca:5984")
+    username = raw_input("Username for [%s]: " % server)
+    password = getpass.getpass("Password for [%s]: " % server)
+    server.resource.credentials = (username,password)
+
+    #find the test_grid database
+    database = server["test_grid"]
```

Comment:
This should use the proper database.  You should get this from the .gangasnoplus config file (which all users with ganga should have and can be read using the examples in GangaSNOplus/Lib/Utilities/RATUtil.py), similarly the DB server should be chosen from the config.  If no config exists, then prompt the user for the server and dbname.